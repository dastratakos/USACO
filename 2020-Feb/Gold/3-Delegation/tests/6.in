841
326 174
307 229
756 638
205 174
649 419
406 174
415 431
378 613
35 272
812 378
820 791
366 832
378 308
393 108
635 537
312 549
451 215
300 664
436 787
333 378
294 682
345 167
377 660
747 310
378 822
807 169
658 402
378 783
614 37
378 346
584 666
260 23
302 71
71 655
174 306
676 149
330 161
420 414
530 153
430 174
166 578
371 92
688 553
718 295
174 201
825 795
328 322
174 593
518 73
99 378
405 575
694 296
8 152
20 423
399 834
174 615
769 371
299 154
656 439
184 166
440 174
777 301
543 700
659 28
48 323
174 612
404 798
174 141
226 96
818 362
802 340
507 641
674 174
786 368
373 259
3 70
6 246
679 192
174 395
483 174
286 174
10 174
378 603
174 299
174 463
61 262
714 429
182 684
174 801
174 107
321 378
1 174
750 174
587 588
594 582
378 756
348 378
592 378
400 339
147 764
135 348
706 416
566 243
174 251
471 27
339 554
685 211
568 786
120 174
141 285
136 197
174 184
736 174
340 424
375 811
552 361
469 380
622 265
198 408
411 435
174 240
139 208
773 378
334 313
151 836
484 378
174 314
75 430
550 213
174 557
472 635
454 753
646 590
60 378
643 499
9 11
331 378
41 69
595 784
174 606
172 115
738 378
532 232
519 75
46 823
379 378
56 174
140 123
185 337
790 378
378 261
388 193
242 114
128 696
246 210
90 622
421 165
760 231
575 60
449 378
378 816
612 830
231 623
187 669
188 174
478 614
793 730
786 581
356 378
704 571
175 817
174 661
271 577
292 573
827 117
378 234
675 174
174 250
304 93
473 378
174 478
233 174
244 717
174 746
153 672
277 50
275 679
479 546
237 312
815 716
464 702
832 631
337 748
13 731
267 282
262 378
208 763
344 222
95 718
652 47
179 36
264 378
174 254
607 176
137 378
298 837
378 421
152 778
256 255
253 482
684 174
72 692
54 174
133 839
442 526
274 124
174 25
174 542
528 174
834 378
620 627
170 378
247 378
540 289
100 147
549 174
641 174
378 785
159 174
534 13
16 658
512 841
68 94
476 639
378 18
764 45
409 586
174 487
149 727
109 65
579 374
589 101
45 347
343 454
660 266
628 735
618 245
118 828
428 375
784 358
740 198
362 174
51 326
711 821
734 325
378 472
146 630
571 174
621 663
822 242
538 174
632 352
444 784
250 443
841 400
633 328
174 26
4 206
839 392
223 378
523 187
670 781
62 130
174 706
732 112
357 225
142 453
474 174
353 406
281 396
712 157
378 164
210 452
207 754
623 224
370 174
636 531
327 539
173 278
384 496
408 324
716 378
26 523
710 465
227 442
274 445
293 378
662 742
597 378
319 774
124 112
487 548
678 335
378 504
195 8
378 677
801 204
329 838
787 34
309 650
686 719
34 550
378 768
570 563
59 331
651 513
702 174
219 163
801 29
639 551
174 765
14 383
278 759
174 544
378 697
529 556
521 597
174 461
677 401
88 378
174 15
77 4
788 411
39 145
378 572
546 297
804 156
726 665
503 302
567 521
423 524
336 319
772 403
296 113
696 413
235 790
748 378
301 19
73 174
755 62
40 174
174 274
687 174
378 81
155 378
673 174
378 407
598 378
19 284
798 185
682 174
378 788
105 378
629 168
457 378
800 180
733 509
497 671
174 366
382 488
221 377
174 676
466 618
619 516
214 601
480 576
178 80
174 711
174 802
193 132
165 570
401 98
197 83
657 194
378 540
369 183
765 126
398 378
63 22
650 134
52 162
98 705
272 520
544 177
378 256
624 174
823 722
427 567
559 726
181 30
731 818
416 39
728 475
378 766
107 125
122 741
391 662
418 357
174 789
23 174
174 780
574 659
378 808
162 786
667 724
378 580
509 698
700 97
517 174
808 248
707 373
683 273
803 477
389 541
703 473
456 440
174 476
698 84
313 76
228 626
249 543
482 174
824 743
426 625
2 437
174 762
723 283
57 409
490 565
117 174
447 782
771 174
654 120
53 760
819 545
461 142
364 391
691 258
838 78
412 378
582 495
378 486
92 150
789 122
378 63
527 378
378 133
289 227
448 209
468 470
441 417
311 809
174 364
713 59
361 52
363 99
414 237
254 751
161 174
491 399
489 813
378 85
7 460
572 320
251 510
422 484
701 41
599 77
739 274
407 600
43 17
365 814
150 378
754 701
174 279
717 595
516 466
744 637
786 715
106 532
176 236
467 378
513 40
310 378
215 507
378 792
174 621
69 88
453 191
828 174
18 376
806 428
378 564
174 20
31 91
506 378
157 500
462 732
580 175
174 793
209 378
551 291
378 498
297 737
721 260
394 174
186 378
555 459
381 174
378 275
174 479
58 713
458 673
192 151
127 807
378 7
378 820
655 457
378 160
168 174
263 79
119 378
378 607
596 530
378 200
258 351
96 690
93 456
290 3
325 247
123 103
183 390
837 438
358 190
671 699
174 761
280 805
15 447
751 770
378 636
717 378
433 604
508 779
627 518
463 410
481 8
66 514
317 569
305 321
316 379
378 806
573 381
196 341
67 268
581 485
378 179
78 174
306 102
752 592
158 668
613 287
445 525
126 111
241 678
268 825
144 611
230 803
515 174
11 360
378 367
238 758
385 119
69 276
342 829
378 241
174 833
615 647
666 494
279 129
743 174
536 216
130 61
745 189
236 72
425 315
725 393
154 214
735 481
791 16
174 468
125 610
17 74
274 652
616 772
378 38
143 309
259 527
203 728
349 378
794 490
367 681
560 32
378 14
222 502
378 796
378 53
257 174
610 24
446 174
220 492
663 384
690 378
542 173
495 2
585 824
83 303
174 136
617 140
557 555
626 48
602 450
742 775
665 680
44 804
174 733
378 127
324 174
261 218
556 589
805 378
138 121
174 810
374 56
378 249
134 378
322 738
318 174
378 87
514 112
630 155
314 31
28 264
206 174
455 568
520 253
548 397
32 511
174 489
424 64
590 174
174 794
611 801
116 170
148 336
562 398
697 221
350 558
174 220
2 239
25 667
486 307
720 536
439 446
315 405
8 605
774 174
82 620
500 174
724 434
799 174
50 579
717 628
813 560
604 378
470 68
243 327
645 135
796 632
174 55
217 174
564 382
70 1
351 535
668 316
42 378
558 226
541 695
174 512
378 281
103 378
194 378
27 144
200 415
189 378
383 101
783 670
499 349
108 812
378 386
79 426
174 389
282 501
431 288
378 388
561 689
501 683
443 338
332 174
465 171
174 686
821 387
452 483
86 422
413 378
400 497
378 469
378 199
835 739
810 298
234 33
699 723
390 159
378 777
681 591
378 372
797 827
76 192
121 334
435 749
174 355
535 174
372 694
212 174
239 370
638 714
729 378
376 9
450 219
101 365
778 181
191 359
397 708
829 598
609 223
216 458
87 344
826 378
757 174
689 378
784 263
419 186
160 786
163 378
378 436
634 42
763 257
780 345
642 776
653 455
5 691
492 300
378 44
47 767
89 584
49 594
252 634
347 174
21 757
323 174
539 174
695 354
577 105
211 378
341 104
708 644
104 449
245 52
270 378
265 12
284 432
709 649
640 721
493 332
174 343
792 67
80 137
631 317
403 174
533 378
753 800
505 202
174 89
637 174
112 547
232 462
522 235
177 43
378 109
386 840
583 21
174 57
248 587
171 674
565 441
174 559
174 688
225 330
378 269
131 290
378 238
545 624
816 508
648 356
378 608
766 616
809 274
831 363
110 609
12 736
295 274
378 267
202 174
586 693
722 233
112 230
576 533
378 203
201 642
672 378
115 174
