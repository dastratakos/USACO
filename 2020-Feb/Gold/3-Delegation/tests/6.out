110010000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
